games/games.js

```diff
@@ -83,29 +95,50 @@ logoutButton.addEventListener('click', () => {
  // on load . . .
 window.addEventListener('', async() => {
```

Comment:
really close, currently this doesn't display the past games on initial load. you're just missing the word load inside the empty strings on line 96.